Classes/CocoaLumberjack.swift

```diff
@@ -16,7 +16,7 @@
 import Foundation
 
 extension DDLogFlag {
-    public static func fromLogLevel(logLevel: DDLogLevel) -> DDLogFlag {
+    public static func fromLogLevel(_ logLevel: DDLogLevel) -> DDLogFlag {
```

Comment:
How about renaming this in `from(_ logLevel: DDLogLevel)` to follow Swift 3 API naming convention?